iroha/src/sumeragi.rs

```diff
@@ -23,6 +24,9 @@ use std::{
 /// Amount of time Peer waits for CommitMessage from the proxy tail.
 pub const COMMIT_TIME: Duration = Duration::from_millis(1000);
 
+/// Amount of time Peer waits for TxReceipt from the leader.
+pub const TX_RECEIPT_TIME: Duration = Duration::from_millis(100);
```

Comment:
We need to provide configuration of such parameters in the same PR they arrive to Iroha.
Please update config.